cmd/build.go

```diff
@@ -12,12 +12,16 @@ import (
 	"os"
 	"strings"
 
+	"github.com/open-policy-agent/opa/bundle"
+
 	"github.com/spf13/cobra"
 
 	"github.com/open-policy-agent/opa/compile"
 	"github.com/open-policy-agent/opa/util"
 )
 
+const defaultPublicKeyID = "opa_default_key"
```

Comment:
For some reason this key ID bothers me--I think it's because `opa` and `key` seem redundant. Could we just use `"default"`?

Reply:
Digging into this a bit more...what am I doing wrong?

```
# build a bunlde
$ opa build . --signing-key=secret --signing-alg=HS256

# serve over http
$ python3 -m http.server
```

Inside another terminal:

```
$ opa run -s --set services.default.url=http://localhost:8000 --set bundles.default.resource=bundle.tar.gz --set keys.opa_default_key.key=secret --set keys.opa_default_key.algorithm=HS256
{"addrs":[":8181"],"diagnostic-addrs":[],"insecure_addr":"","level":"info","msg":"Initializing server.","time":"2020-07-06T17:51:53-04:00"}
{"level":"info","msg":"Starting bundle downloader.","name":"default","plugin":"bundle","time":"2020-07-06T17:51:53-04:00"}
{"level":"error","msg":"Bundle download failed: verification key ID is empty","name":"default","plugin":"bundle","time":"2020-07-06T17:51:53-04:00"}
```

It seems the JWTs produced by `opa build` do not have the key ID claim contained in them. This seems like a bug?

Reply:
Updated the default key ID to `default`. 

The key ID can be optionally included in the JWT by providing a path to the JSON file that contains optional claims (`--claims-file`). OPA will look for the key ID on the command-line first, then in the OPA config and finally in the JWT. We've documented this [here](https://deploy-preview-2475--openpolicyagent.netlify.app/docs/edge/management/#signature-format).


```bash
# build a bundle
$ opa build . --signing-key=secret --signing-alg=HS256

# serve over http
$ python3 -m http.server

# specify the key ID on the command-line 
$ opa run -s --set services.default.url=http://localhost:8000 --set bundles.default.resource=bundle.tar.gz --set bundles.default.signing.keyid=default --set keys.default.key=secret --set keys.default.algorithm=HS256
{"addrs":[":8181"],"diagnostic-addrs":[],"insecure_addr":"","level":"info","msg":"Initializing server.","time":"2020-07-06T16:12:50-07:00"}
{"level":"info","msg":"Starting bundle downloader.","name":"default","plugin":"bundle","time":"2020-07-06T16:12:50-07:00"}
{"level":"info","msg":"Bundle downloaded and activated successfully.","name":"default","plugin":"bundle","time":"2020-07-06T16:12:50-07:00"}
```

**claims.json**
```json
{
    "keyid": "default"
}
```

```bash
# build a bundle
$ opa build . --signing-key=secret --signing-alg=HS256 --claims-file claims.json

# serve over http
$ python3 -m http.server

# use key ID specified in the JWT
$ opa run -s --set services.default.url=http://localhost:8000 --set bundles.default.resource=bundle.tar.gz --set keys.default.key=secret --set keys.default.algorithm=HS256
{"addrs":[":8181"],"diagnostic-addrs":[],"insecure_addr":"","level":"info","msg":"Initializing server.","time":"2020-07-06T16:22:44-07:00"}
{"level":"info","msg":"Starting bundle downloader.","name":"default","plugin":"bundle","time":"2020-07-06T16:22:44-07:00"}
{"level":"info","msg":"Bundle downloaded and activated successfully.","name":"default","plugin":"bundle","time":"2020-07-06T16:22:44-07:00"} 
```

Reply:
@ashutosh-narkar WDYT about making `opa build` set the keyid claim automatically (the claim file could override it if needed).

Reply:
Just to close the loop on the offline conversation. We reached a decision to have `opa build` inject the keyid claim using the default value, by default.

Reply:
Done.

```bash
# build a bundle
opa build -b . --signing-key=secret --signing-alg=HS256

# serve over http
$ python3 -m http.server

# run OPA
opa run -s --set services.default.url=http://localhost:8000 --set bundles.default.resource=bundle.tar.gz  --set keys.default.key=secret --set keys.default.algorithm=HS256
{"addrs":[":8181"],"diagnostic-addrs":[],"insecure_addr":"","level":"info","msg":"Initializing server.","time":"2020-07-09T01:33:30-07:00"}
{"level":"info","msg":"Starting bundle downloader.","name":"default","plugin":"bundle","time":"2020-07-09T01:33:30-07:00"}
{"level":"info","msg":"Bundle downloaded and activated successfully.","name":"default","plugin":"bundle","time":"2020-07-09T01:33:30-07:00"}
```